packages/@okta/vuepress-site/docs/guides/saml-inline-hook/main/index.md

```diff
@@ -0,0 +1,327 @@
+                            "value": [
+                                {
+                                  "op": "add",
+                                  "path": "/claims/extPatientId",
+                                  "value": {
+                                      "attributes": {
+                                      "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"    
+                                        },
+                                       "attributeValues":[
+                                       {
+                                       "attributes": {
+                                         "xsi:type": "xs:integer"
+                                       },
+                                        "value": patientID,
+                                       }
+                                       ]
+                                  }
+                                }
+                            ]
+                          }
+                      ]
+                    }
+
+  console.log("Added patient ID claim to SAML assertion: " + returnValue.commands[0].value[0].value.attributeValues[0]["value"]);
+
+  response.send(JSON.stringify(returnValue));
+  }
+  else {
+
+  console.log("Not a patient. No change to SAML assertion.");
+  response.status(204).send();
+  }
+
+}
+);
+```
+
+## Activate and enable the SAML assertion inline hook
+
+The SAML assertion inline hook must be activated and enabled within your Okta Admin Console.
+
+Activating the SAML assertion inline hook registers the hook with the Okta org and associates it with your external service. Enabling the SAML assertion inline hook associates the hook with your SAML application.
+
+### Activate the SAML assertion inline hook
+
+1. Navigate to the **Workflow** > **Inline Hooks** page.
+
+1. Click **Add Inline Hook** and select **SAML** from the dropdown menu.
+
+1. Add a name for the hook (in this example, "Patient SAML Hook").
+
+1. Add your external service URL, including the endpoint. For example, use your Glitch project name with the endpoint: `https://your-glitch-projectname.glitch.me/SAMLHook`.
+
+1. Include authentication field and secret. In this example:
+
+   * **Authentication field** = `authorization`
+   * **Authentication secret** = `Basic YWRtaW46c3VwZXJzZWNyZXQ=`
+
+1. Click Save.
+
+The SAML assertion inline hook is now set up with a status of active.
+
+### Enable the SAML assertion inline hook
+
+1. Navigate to **Applications** and select your SAML application (in this example, "Spring Book SAML").
+
+1. Click the **General** tab.
+
+1. From the **SAML Settings** tile, click **Edit**.
+
+1. Click **Next** to get to the **Configure SAML** tab.
+
+1. From the **SAML Settings, General** heading, click **Show Advanced Settings**.
+
+1. In the **Assertion Inline Hook** field, select your registered inline hook from the dropdown menu (in this example, "Patient SAML Hook").
+
```

Comment:
Should we include the steps req'd to save the work? My concerns is that a user enters this info then navigates away.

Reply:
Agree. Added one more step: Click **Next** and **Finish**.